cmd/node/config/enableEpochs.toml

```diff
@@ -250,13 +250,16 @@
 
     # BLSMultiSignerEnableEpoch represents the activation epoch for different types of BLS multi-signers
     BLSMultiSignerEnableEpoch = [
-        { EnableEpoch = 0, Type = "no-KOSK"},
-        { EnableEpoch = 1, Type = "KOSK"}
+        { EnableEpoch = 0, Type = "no-KOSK" },
+        { EnableEpoch = 1, Type = "KOSK" }
     ]
 
     # SetGuardianEnableEpoch represents the epoch when the guard account feature is enabled in the protocol
```

Comment:
can we move L257 - L262 after the `BLSMultiSignerEnableEpoch` block?

Reply:
moved